wallet/wallet.go

```diff
@@ -498,6 +498,11 @@ out:
 				continue
 			}
 			timeout = req.lockAfter
+			if timeout == nil {
+				log.Info("Wallet unlocked without timeout")
+			} else {
+				log.Info("Wallet temporarily unlocked")
```

Comment:
There's no reasonable value to pass in as a timeout for extra logging information from calling code similar to the grpc server.  There is no set timeout, it simply locks the wallet as soon as it no longer needs it.